packages/gestalt/src/Layout.css

```diff
@@ -83,6 +83,20 @@
   left: 0;
 }
 
+/* size */
+
+.small {
+  min-height: 36px;
```

Comment:
Reason for using `min-height` opposed to `height` is multiline:

**min-height**
![image](https://user-images.githubusercontent.com/127199/39454121-4bf2c4e2-4c8e-11e8-85d5-49c9d023f195.png)

**height**
![image](https://user-images.githubusercontent.com/127199/39454131-5dc7ebb6-4c8e-11e8-9d31-b76495a9d7e4.png)